src/content/app/entity-viewer/variant-view/transcript-consequences/TranscriptConsequences.tsx

```diff
@@ -88,24 +130,64 @@ const TranscriptConsequences = (props: Props) => {
     );
   }
 
+  type geneDataWithTranscriptConsequencesType = GeneInResponse & {
+    transcriptConsequences: transcriptConsequencesWithTranscriptType;
+  };
```

Comment:
Style:
- same comments as for the type above
- plus, please don't declare types inside of functions

Reply:
same thing as before. I am leaving the `Type` suffix as it is.